pkg/registry/batch/job/strategy_test.go

```diff
@@ -36,9 +36,10 @@ import (
 
 func TestJobStrategy(t *testing.T) {
 	cases := map[string]struct {
-		ttlEnabled        bool
-		indexedJobEnabled bool
-		suspendJobEnabled bool
+		ttlEnabled                bool
+		indexedJobEnabled         bool
+		suspendJobEnabled         bool
+		jobTrackingWithFinalizers bool
```

Comment:
Nit: let's keep the naming consistent, how about `trackingWithFinalizersEnabled`?